### DISCUSSION TITLE
Using context (express-graphql) shield middleware doesn't work

### DISCUSSION THREAD

**Discussion by mittalyashu** [maticzav/graphql-shield#1039] [Q&A]
I am using `express-graphql` package for creating graphql server.

```js
app.use(
  "/graphql",
  graphqlHTTP({
    schema: schemaMiddleware,
    graphiql: process.env.NODE_ENV === "development",
    context: ({ req }) => {
      console.log("context req");
      console.log(req);
    }
  })
);
```

And while adding `graphql-shield` via graphql middleware.
It logs this to terminal

```
[Function: context] { _shield: { cache: {} } }
```

My use-case is the pass the user data using context to graphql-shield middleware.

**Comment by maticzav** [comment]
Could you create a small reproduction repository or CodeSandbox?
